### DISCUSSION TITLE
Feb 27

### DISCUSSION THREAD

**Discussion by oauch** [GymHubCommunity/GymHub-FE#242] [DailyMeeting]
### ⚽️ 회의 내용

개발 현황
슬희 - 마이페이지 / 달에 해당하는 운동기록 조회
윤혁 - 운동 추가하기 기능 구현하다 사망..
수영 - 해시태그 검색 / 전체 탭

20시 회의

### 🏀 오늘 할 일

슬희 - 충돌 해결 후, 스냅샷 예정
윤혁 - 운동 추가하기 기능 및 API
수영 - 최종 컨펌

**Comment by hanseulhee** [comment]
끝이 보인다 .... 보여 ,,,,,,
